client/src/app/shared/components/agenda-content-object-form/agenda-content-object-form.component.html

```diff
@@ -1,4 +1,4 @@
-<ng-container *osPerms="'agenda.can_manage'">
+<ng-container *osPerms="'can_manage'">
```

Comment:
Are you sure that this is correct?

I suggest to make `Permission` public in the BaseComponent, so this enum is available in every template.